src/states/elder/mod.rs

```diff
@@ -125,15 +125,14 @@ impl Elder {
         let public_id = *full_id.public_id();
         let connection_info = network_service.our_connection_info()?;
         let p2p_node = P2pNode::new(public_id, connection_info);
-        let mut first_ages = BTreeMap::new();
-        let _ = first_ages.insert(public_id, MIN_AGE_COUNTER);
+        let mut ages = BTreeMap::new();
+        let _ = ages.insert(public_id, MIN_AGE_COUNTER);
```

Comment:
What would be the consequences of this failing

Reply:
Not sure what you mean by failing, but we can safely ignore the return value of `insert` because we just created the collection one line above so it's always going to return `None`.